pkg/syncer/metadatasyncer.go

```diff
@@ -343,20 +343,31 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 					cnsoperatorv1alpha1.CnsStoragePolicyQuotaSingular, err)
 			}
 
+			isStorageQuotaM2Enabled := metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.StorageQuotaM2)
+			go func() {
+				if isStorageQuotaM2Enabled {
+					// TODO: Wait for fullSync to create CnsVolumeInfo CRs if necessary
```

Comment:
When are you planning to add this logic?

Reply:
Separate PR